test/integration/tasklist_page_test.rb

```diff
@@ -20,28 +20,38 @@ class TasklistPageTest < ActionDispatch::IntegrationTest
       assert_equal "Learn to drive a car: step by step", component_args[:title]
     end
 
-    assert page.has_css?(shared_component_selector('lead_paragraph'))
-
-    within_static_component 'lead_paragraph' do |paragraph_args|
-      assert_equal "Check what you need to do to learn to drive.", paragraph_args[:text]
-    end
-
     assert page.has_css?(shared_component_selector('task_list'))
 
     within_static_component('task_list') do |tasklist_args|
       assert_equal 6, tasklist_args[:groups].count
 
-      assert_equal [], tasklist_step_keys(tasklist_args) - %w(title panel panel_descriptions panel_links)
+      assert_equal [], tasklist_step_keys(tasklist_args) - %w(title contents)
 
-      assert_equal [], tasklist_panel_links_keys(tasklist_args) - %w(href text), []
+      assert_equal [], tasklist_content_keys(tasklist_args) - %w(type text style links), []
```

Comment:
Do we need the trailing `, []` on this line and the line below?  It looks like a mistake that sneaked through the last review